public/modules/custom/hel_tpm_search_autosuggest/js/hel-tpm-search-autosuggest.js

```diff
@@ -140,6 +140,9 @@
             Drupal.behaviors.hel_tpm_search_autocomplete.showHideAutocomplete(this, context);
           });
 
+        $(form).closest('form').on('submit', function(e) {
```

Comment:
What are these changes? Could you add them to the testing instructions?

Reply:
Whoops wrong branch.. Moved these changes to another PR